lambdas/example-lambda/handler.ts

```diff
@@ -8,36 +9,39 @@ import { ConnectionManager } from "../../shared/utils/connection-manager";
 import { ExampleModel } from "../../shared/models/example.model";
 import { v4 } from "uuid";
 import { createConfig } from "./config";
+import { logIncomingEvent } from "../../shared/middleware/log-incomming-event";
 
 const config = createConfig(process.env);
 
-export async function handle(event: any, _: Context): Promise<any> {
-  try {
-    const query = event.queryStringParameters;
-    const { exampleParam } = joi.attempt(query, schema, { abortEarly: false });
+export const handle = middy(
+  async (event: APIGatewayEvent, _context: Context): Promise<APIGatewayProxyResult> => {
+    try {
+      const query = event.queryStringParameters;
+      const { exampleParam } = joi.attempt(query, schema, { abortEarly: false });
 
-    winstonLogger.info(`Hello from ${config.appName}. Example param is: ${exampleParam}`);
+      winstonLogger.info(`Hello from ${config.appName}. Example param is: ${exampleParam}`);
 
-    const connectionManager = new ConnectionManager();
-    const connection = await connectionManager.getConnection();
+      const connectionManager = new ConnectionManager();
+      const connection = await connectionManager.getConnection();
 
-    await connection.getRepository(ExampleModel).save(
-      ExampleModel.create({
-        id: v4(),
-        email: "some@tmp.pl",
-        firstName: "Test",
-        lastName: "User",
-      }),
-    );
+      await connection.getRepository(ExampleModel).save(
+        ExampleModel.create({
+          id: v4(),
+          email: "some@tmp.pl",
+          firstName: "Test",
+          lastName: "User",
+        }),
+      );
 
-    return awsLambdaResponse(200, {
-      success: true,
-      data: await connection.getRepository(ExampleModel).find({}),
-    });
-  } catch (e) {
-    // eslint-disable-next-line no-console
-    console.error(e);
+      return awsLambdaResponse(200, {
+        success: true,
+        data: await connection.getRepository(ExampleModel).find({}),
+      });
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
```

Comment:
Let's remove that console and use our logger here

Reply:
oh i haven't seen that, but yea, definitely

Reply:
done